homeassistant/components/salus/__init__.py

```diff
@@ -0,0 +1,54 @@
+"""Initialize Salus integration."""
+import logging
+
+from salus.api import Api
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_DEVICE, CONF_PASSWORD, CONF_USERNAME
+from homeassistant.helpers.typing import HomeAssistantType
+
+from .const import DOMAIN, PLATFORMS
+from .coordinator import SalusDataUpdateCoordinator
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup(hass: HomeAssistantType, config: dict) -> bool:
+    """Set up the Salus integration."""
+    hass.data.setdefault(DOMAIN, {})
+
+    # Return boolean to indicate that initialization was successful.
+    return True
+
+
```

Comment:
This is no longer needed, moved the setdefault to the async_setup_entry. 
For this change, it might be needed to rebase onto the latest dev.


```suggestion
```